src/rust/aggregator/src/pipelines/update_materialized_view.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::anyhow;
+use chrono::{DateTime, Duration, Utc};
+use sqlx::PgPool;
+
+use aggregator::{Pipeline, PipelineAggregationResult, PipelineError};
+
+pub struct UpdateMaterializedView {
```

Comment:
Before we lock this into the codebase, how about we go with `RefreshMaterializedView` , to make it clear that we are doing a 1-to-1 PostgreSQL operation?

Reply:
:+1: